website/docs/r/cluster.html.markdown

```diff
@@ -307,11 +307,13 @@ Physical location of the region.
 
 -> **NOTE:** Prior to setting these options please ensure you read https://docs.atlas.mongodb.com/cluster-config/additional-options/.
 
+-> **NOTE:** This argument has been changed to type list make sure you have the proper syntax.
```

Comment:
Just add a note that it is a list with only one element (MaxItems: 1).

Reply:
done!